backend/de.metas.report/metasfresh-report-service/src/main/java/de/metas/report/jasper/OrgLogoClassLoaderHook.java

```diff
@@ -65,13 +65,17 @@ public static OrgLogoClassLoaderHook newInstance()
 	//
 	// Org Logo resource matchers
 	private static final String SYSCONFIG_ResourceNameEndsWith = "de.metas.adempiere.report.jasper.OrgLogoClassLoaderHook.ResourceNameEndsWith";
-	private static final String DEFAULT_ResourceNameEndsWith = "de/metas/generics/logo.png";
+	private static final String DEFAULT_ResourceNameEndsWith = "de/metas/generics/logo.png;";
 	private final ImmutableSet<String> resourceNameEndsWithMatchers;
-
+	
+	private static final String otherResourceNameEndsWith = "de/metas/generics/ReportBottom_Logo_ID.png";
```

Comment:
```suggestion
	private static final String reportBottomLogoResourceEndsWith = "de/metas/generics/ReportBottom_Logo_ID.png";
```
...please use clear variable and column names. also for the constant and AD_Sysconfig-name

Reply:
why this is needed?
basically that's totally against of what we discussed: a way to use AD_OrgInfo.MyLogoImage_ID without changing the code.